<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWaypointDialog</class>
 <widget class="QDialog" name="UserWaypointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Edit Position</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="7" column="2">
    <widget class="QLineEdit" name="lineEditRouteUserWaypointLatLon">
     <property name="toolTip">
      <string>Recognized coordinate formats:
- Degrees, minutes and seconds : N49° 26' 41.57&quot; E9° 12' 5.49&quot; or 49° 26' 41.57&quot; N 9° 12' 5.49&quot; E
- Degrees and minutes: N54* 16.82' W008* 35.95', N 52 33.58 E 13 17.26 or 49° 26.69' N 9° 12.09' E
- Degrees only: 49.4449° N 9.2015° E or N 49.4449° E 9.2015°
- Degrees with sign: 49.4449 9.2015. Negative is west or south. Order for latitude and longitude depends on option settings.
Degree, minute and second signs can be omitted for above formats.
Latitude has to be first and N/S and E/W designators are required.
A space is only needed to separate latitude and longitude or degrees/minutes/seconds if no signs used.
Other formats:
- Degrees and minutes: N44124W122451, N14544W017479 or S31240E136502
- Degrees only 46N078W
- Degrees and minutes 4620N07805W
- Degrees, minutes and seconds 481200N0112842E (SkyVector)
- Degrees and minutes in pair &quot;N6500 W08000&quot; or &quot;N6500/W08000&quot;
- NAT type 5020N
Formats above have to be given exactly as shown in the examples.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="labelRouteUserWaypointHeader2">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lineEditRouteUserWaypointIdent">
     <property name="toolTip">
      <string>Required ident of the waypoint.
A short all upper case ident without spaces is recommended.

Idents starting with &quot;WP&quot; followed by a number are automatically renamed to reflect flying order.
Examples: &quot;WP1&quot;, &quot;WP2&quot; or &quot;WP10&quot;.</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="labelRouteUserWaypointCoordStatus">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointLatLon">
     <property name="text">
      <string>&amp;Latitude and Longitude:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypointLatLon</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="labelRouteUserWaypointHeader">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBoxRouteUserWaypoint">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="lineEditRouteUserWaypointRegion">
     <property name="toolTip">
      <string>Optional two letter ICAO region of the waypoint.</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointName">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypointName</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="lineEditRouteUserWaypointName">
     <property name="toolTip">
      <string>Waypoint name</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointIdent">
     <property name="text">
      <string>&amp;Ident:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypointIdent</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointRegion">
     <property name="text">
      <string>&amp;Region:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditRouteUserWaypointRegion</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelRouteUserWaypointComment">
     <property name="text">
      <string>&amp;Remarks:</string>
     </property>
     <property name="buddy">
      <cstring>plainTextEditRouteUserWaypointComment</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPlainTextEdit" name="plainTextEditRouteUserWaypointComment">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditRouteUserWaypointIdent</tabstop>
  <tabstop>lineEditRouteUserWaypointRegion</tabstop>
  <tabstop>lineEditRouteUserWaypointName</tabstop>
  <tabstop>plainTextEditRouteUserWaypointComment</tabstop>
  <tabstop>lineEditRouteUserWaypointLatLon</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
